readme.txt

```diff
@@ -1,12 +1,53 @@
-=== inseri core ===
-Contributors: inseri swiss
-Tags: block, inseri
-Requires at least: 5.8
-Requires PHP: 7.0
+=== Scientific and Interactive Blocks - inseri core ===
+Contributors: inseriswiss
+Tags: Gutenberg blocks, WordPress blocks, gutenberg, blocks, inseri
```

Comment:
Add `community plugin` as a tag (OSS is implied by being an official WordPress plugin and by the License, I guess). In case they have any taxonomy use it to denote that there is no premium/freemium/commercial product associated with it.

Reply:
I don't see any usage of `community plugin`,
https://wordpress.org/plugins/tags/community-plugin/ (observe url bar, how it is redirect)
whereas `gutenberg blocks` is used  https://wordpress.org/plugins/tags/gutenberg-blocks/